### DISCUSSION TITLE
27 August Dev Session

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1400] [General]
When: 27 August---- 2000 UTC+8/1300 CET/1200 GMT/0800 ET

Where: [via Matrix](https://matrix.to/#/#mesa-dev:matrix.org)

Proposed Agenda:
- NUMFOCUS Grant
- Mesa-Geo
- SFI Tutorial

@jackiekazil @rht @Corvince @wang-boyu @aiwhoo @nunezco2 @EwoutH Please feel free to add or takeaway from this agenda

**Comment by rht** [comment]
Adding also @gehirndienst. Context: https://github.com/projectmesa/mesa-geo/discussions/94.
